test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8411,108 +8436,102 @@ def is_history_enabled(self):
             return True
         return False
 
-    def cleanup_jobs(self, extend=None, runas=None):
+    def cleanup_jobs(self):
```

Comment:
why did you remove extend and runas fields from this function? What if some test wants to do deletehistory as part of cleaning up jobs?

Reply:
> why did you remove extend and runas fields from this function?

It was useless after new changes, in this PR we made cleanup_jobs to clean jobs at root user with nomail, force, deletehistory (if applicable).



> What if some test wants to do deletehistory as part of cleaning up jobs?

cleanup_jobs was never used extend passed to it (even before changes to this PR) and now also not using since it is passing its own extend (nomail, force, deletehistory) to select() and deljob().